assets/js/weatherApp.js

```diff
@@ -0,0 +1,44 @@
+let weather = {
+    "apikey": "375090722ff9b13931a9ed44d6f32f14",
```

Comment:
public apikey will be banned. you need to move it into ENV variable and enable weather feature only if apikey setted